### PR TITLE
Encoding.createUrl method returns url with duplicate slash

### PR DESCRIPTION
**Pull request by krimpedance**
When set `baseUrlString` to `http://www.example.com/` and `urlString` to `/path`, `createUrl` returns `https://www.example.com//path`.

### CODE REVIEW

File: fuel/src/test/kotlin/com/github/kittinunf/fuel/EncodingTest.kt [hunk 1]
```diff
@@ -21,6 +21,18 @@ class EncodingTest : BaseTestCase() {
...
+        val request = Encoding(
+                httpMethod = Method.GET,
+                baseUrlString = "http://www.example.com/",
+                urlString = "test",
+                parameters = listOf("a" to "b")
+        ).request
+
+        assertThat(request.url.toString(), isEqualTo("http://www.example.com/test?a=b"))
+    }
+
```
Comment (kittinunf) [inline fuel/src/test/kotlin/com/github/kittinunf/fuel/EncodingTest.kt]: Should we add a test for the problem that you face, and see whether the code fixes or not?

```
baseUrlString = "http://www.example.com/"
urlString = "/test"
```

Comment (krimpedance) [reply fuel/src/test/kotlin/com/github/kittinunf/fuel/EncodingTest.kt]: Yes, please.

Comment (kittinunf) [reply fuel/src/test/kotlin/com/github/kittinunf/fuel/EncodingTest.kt]: Could you please add them? 😄

Comment (krimpedance) [reply fuel/src/test/kotlin/com/github/kittinunf/fuel/EncodingTest.kt]: @kindraywind 
Sure.
There was a mistake in adding test.

**Comment by kittinunf** [comment]
I will update the branch and try to merge. Thanks @krim

**Pull request merged**